src/components/Statusbar.js

```diff
@@ -0,0 +1,12 @@
+var React = require('react');
+import NumberWidget from './widgets/NumberWidget';
+
+export default class Statusbar extends React.Component {
+  render() {
+    return <div id='statusbar'>
```

Comment:
also add `className='statusbar'` so we don't have to do a bunch of `!important`s in the stylesheet